scripts/plugins/logging.py

```diff
@@ -39,6 +39,8 @@ def mask_message(message):
     """
     if message is None:
         return message
+    if BITOPS_logging_masks is None:
+        return message
```

Comment:
How about specifying the right default for the empty `mask`? I believe it's a list by default.

Maybe adding `else []` here would save us from future issues?
https://github.com/bitovi/bitops/blob/d5f625eca3effcd27741afeb703c72a54bb3039d/scripts/plugins/settings.py#L126-L130

WDYT?

Reply:
Additionally, looks like we've missed adding new `mask` to bitops schema here:
https://github.com/bitovi/bitops/blob/main/bitops.schema.yaml

Reply:
I disagree - personally if the value is "" then it should be a none type. None type checking is fast and accurate. Variables being half set isn't a great pattern.

Reply:
Yeah, I understand what you mean.
I just see similar errors in so many places as we're fixing those.

For instance, if schema says `bitops: ` is an `object`, - yeah after parsing yaml technically empty value is `None`, but it's much better for us to consider it as an empty object or `{}` dict.

If schema says `mask:` is a list, technically empty value is `None`, but it's much better to consider it as an empty list `[]`.

And because we have a schema with default types it makes more sense to justify it: https://github.com/bitovi/bitops/blob/d5f625eca3effcd27741afeb703c72a54bb3039d/bitops.schema.yaml#L40-L45
So we're casting to the default type according to the schema that we defined previously.

Technically relying on `None` gives us nothing, but relying on the default schema type is much more beneficial saving us from all the type & empty check issues.